dpp/src/util/json_value.rs

```diff
@@ -273,6 +278,27 @@ impl JsonValueExt for JsonValue {
             _ => bail!("the Json Value isn't a map: {:?}", self),
         }
     }
+
+    fn replace_base64_paths<'a>(
```

Comment:
I just realized that there is a bug in line https://github.com/dashevo/rust-dpp/blob/0183586f75d421e0c7db30b268a0f392cc870680/dpp/src/util/json_value.rs#L360.  It should be `Base64` instead of `Base58` .  After the change, the method `replace_binary_paths()` with `ReplaceWith::Bytes` param could be used.